entrypoint.sh

```diff
@@ -117,7 +117,7 @@ for i in "${!REPOS[@]}"; do
 done
 
 # yumdownloader params
-YUMD_PARAMS="--archlist=x86_64 --archlist=noarch --releasever=${VERSION_ID} ${REPOLIST}"
+YUMD_PARAMS="--no-xattrs --archlist=x86_64 --archlist=noarch --releasever=${VERSION_ID} ${REPOLIST}"
```

Comment:
Ah sorry I meant "it'd be nice if that existed", it doesn't today.  I tested the workaround from https://github.com/openshift/okd/issues/566#issuecomment-802147697 locally, basically:

```
yumdownloader ...
for f in *.rpm; do attr=$(getfattr -m 'user.*' $f | grep -Ee '^user'); setfattr -x ${attr} $f; done
```